src/.htaccess

```diff
@@ -0,0 +1,23 @@
+Options +FollowSymLinks
+
+<Files "login.json">  
+  Require all denied
+</Files>
+
+<Files "config.php">  
+  Require all denied
+</Files>
+
+<Directory "u/">
+  Require all denied
+</Directory>
+
+RewriteEngine On
+RewriteBase /
+
+RewriteRule css/(.*) css/$1
+RewriteRule js/(.*) js/$1
+
+RewriteCond %{REQUEST_FILENAME} !-f  
+RewriteCond %{REQUEST_FILENAME} !-d  
+RewriteRule .* rewrite.php [L] 
```

Comment:
although i use nginx too, perhaps its worth writing some matching nginx rules

Reply:
Sounds good, I was thinking about doing this too. The only problem with this is that NGINX doesn't support local configuration files, like Apache's `.htaccess`. Users would have to add some lines to their global server config. However, I'll still try to figure out what that config could look like.